src/foundry/client/pixi/extensions/polygon.d.mts

```diff
@@ -123,20 +114,31 @@ declare module "pixi.js" {
   }
 
   namespace Polygon {
+    /**
+     * @privateRemarks Foundry uses this type instead of full `ClipperLib.IntPoint` objects
+     */
     interface ClipperPoint {
       X: number;
       Y: number;
     }
 
     /** @internal Helper type for interface to simplify InexactPartial usage */
     type _IntersectClipperOptions = InexactPartial<{
-      /** The clipper clip type */
-      clipType: number;
-
-      /** A scaling factor passed to Polygon#toClipperPoints to preserve precision */
+      /**
+       * The clipper clip type
+       * @remarks ClipperLib functions require a `ClipperLib.ClipType` value, but the Foundry functions have `??=` guards for this.
+       * @defaultValue `ClipperLib.ClipType.ctIntersection`
+       */
+      clipType: ClipperLib.ClipType | null;
+
+      /**
+       * A scaling factor passed to Polygon#toClipperPoints to preserve precision
+       * @defaultValue `1`
+       */
       scalingFactor: number;
     }>;
 
+    /** @remarks `scalingFactor` can't be null as it only has defaults provided by `{scalingFactor=1}` where it has any */
```

Comment:
I'd put this on `scalingFactor` directly. Maybe it doesn't show up as well in intellisense? If so I guess we can leave it.

Reply:
This would be the traditional place for leaving a "why no NullishProps" comment, which is why it got left there, but it doesn't really make sense anymore. In places I used this pattern after this I have just been including the NP/IP reasoning in an `@remarks` in the same block as the `@internal`, which is probably *not* ideal for intellisense tbf. I don't know what the least bad option here is.